### ISSUE TITLE
Add "Big Picture Mode"

### ISSUE THREAD

**Issue by shazzaam7** [xenia-manager/xenia-manager#53]
Add something similar to the Steam's "Big Picture Mode".

**Comment by jaylex32** [comment]
yes please, or a fullscreen button because i have over 40 games and the window is too small, it needs a fullscreen and a resize button.

**Comment by JayTheTechGuy** [comment]
So far I'm loving this app.....didn't care about xenia until now. Dude this would be amazing to use with a controller and use the guide button to bring up a pause menu to exit the game.

**Comment by shazzaam7** [comment]
Update on this issue: For now I'll adapt the app so it can be maximized to Fullscreen because of lack of support for controlling the UI using the controller.

**Comment by jaylex32** [comment]
> Update on this issue: For now I'll adapt the app so it can be maximized to Fullscreen because of lack of support for controlling the UI using the controller.

Look at my fork and tell me if you think is worth to merge it and go for it, thanks

**Comment by shazzaam7** [comment]
It has potential, but since the formatting of the code (Xaml files mostly) is different, I can't just merge it. 
I will have to work on the contribution guide so if someone wants to implement something they can follow it and we can be on the same page. 
That being said, you gave me some ideas about how I might do it (from looking at the screenshots).

**Comment by shazzaam7** [comment]
Just a sidenote: Don't be afraid to do a pull request if you have something that you would want to implement. I'm open to people contributing in the project. I'll either merge it if it's a good idea or if it has some formatting issues implement it myself and credit you.

**Comment by SRC267** [comment]
I use my W11 desktop with 175% display scale, so the XM window gets cut off. So FC of BPM will be a great solution. Right now I simply move the window but it's not ideal lol. My setup is on the couch, so it's a little inconvenient with my mini keyboard/touchpad. On my handheld also, the touch screen and full controller menu support will help.

**Comment by shazzaam7** [comment]
> I use my W11 desktop with 175% display scale, so the XM window gets cut off. So FC of BPM will be a great solution. Right now I simply move the window but it's not ideal lol. My setup is on the couch, so it's a little inconvenient with my mini keyboard/touchpad. On my handheld also, the touch screen and full controller menu support will help.

I think this Fullscreen option I'm working on will help you with 175% display scale. I'll let you know when I'm done with it so you can properly test it on your own setup and give me feedback

For your handheld, controller support is for now scratched because of lack of support in .NET and C# for it. Worst case scenario is I make another app that will act like Big Picture Mode in something that supports controller input better or I create something from scratch. I also don't have any Windows devices that have touchscreen.

**Comment by SRC267** [comment]
> @SRC267 Check out the latest [experimental](https://github.com/xenia-manager/xenia-manager/releases/tag/experimental) and let me know if Fullscreen mode works better for you. (You only need to put the app once in Fullscreen and it will keep the current mode even if you close it and re open it again)

Works as expected in FCM, although it still cuts the top panel if I open it in ![windowed mode](https://github.com/user-attachments/assets/92e6b055-2ced-4873-8495-527991bf116f), I suppose that's just the scaling that's causing it. ![Here's FC](https://github.com/user-attachments/assets/9b31b420-8f15-418b-a95e-653e5ab0de56)

One issue I have in particular for this PC is I'm unable to select ![English as the language](https://github.com/user-attachments/assets/22fc72ba-8a6e-4682-8c56-584e69f30fad), but have no issues on my other systems. I've tried changing it in different window modes and scaling. It could be that it's unable to detect an English language pack, since I've installed W11 on this system without TPM, I selected international English upon Install. Not a big deal since the default language still is showing as English.
